docs/website/docs/overview/configure.md

```diff
@@ -190,7 +190,6 @@ Options here are mostly used for debugging and testing `odo` behavior.
 | `DEVFILE_PROXY`            | Integration tests will use this address as Devfile registry instead of `registry.stage.devfile.io`                                                                                                                                                                                                                                                                             | v3.0.0-beta3  | `my-registry.example.com`       |
 | `TELEMETRY_CALLER`         | Caller identifier passed to [telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md). Case-insensitive. Acceptable values: `vscode`, `intellij`, `jboss`.                                                                                                                                                                                                  | v3.1.0        | `intellij`                      |
 | `ODO_TRACKING_CONSENT`     | Useful for controlling [telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md). Acceptable values: `yes` ([enables telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md) and skips consent prompt), `no` (disables telemetry and consent prompt). Takes precedence over the [`ConsentTelemetry`](#preference-key-table) preference. | v3.2.0        | `yes`                           |
-| `ODO_EXPERIMENTAL_MODE`    | Whether to enable experimental features. See [Experimental Mode](../user-guides/advanced/experimental-mode) for more details. Acceptable values: boolean values<sup>(1)</sup>                                                                                                                                                                                                  | v3.3.0        | `true`                          |
```

Comment:
Can you comment this instead of completely removing it?

Reply:
ok

Reply:
I think it might be because you renamed the experimental-mode.md file. I would remove the link like you've done at other places.

Reply:
The comment is not recognized as expected: 
![comment-odo-exp](https://user-images.githubusercontent.com/9973512/223083955-52b0883d-fbde-46fc-92ab-6e63686a58ae.png)

Reply:
I'm reverting to remove completely the line. The commit is accessible if we want to revert the deletion